src/Lykke.Common.Validation/PasswordHash/PasswordHashAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace Lykke.Common.Validation.PasswordHash
+{
+    /// <summary>
+    ///     Validates string is a valid SHA-256 hash.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class PasswordHashAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            var passwordHash = (string) value;
```

Comment:
Is this safe to cast to string without checking?

Reply:
Refactored to:
```csharp
 if (!(context.PropertyValue is IFormFile file))
```